### ISSUE TITLE
Vérification démarche et test page d'aide Jimsmooth 

### ISSUE THREAD

**Issue by JimAmel** [jimlab/jimlab#16]
https://github.com/jimlab/jimlab/blob/master/macros/jimsmooth.sci

**Comment by sgougeon** [comment]
Pour la page d'aide, devant être conforme / traduisant le cahier des charges en terme d'usages : 
Voir dans le ticket #14 ici : https://github.com/jimlab/jimlab/issues/14#issuecomment-300754759

**Comment by sgougeon** [comment]
Mes remarques concernant le code existant de jimsmooth() sont incluses au PDF joint.
J'examinerai le code de jimsmooth_mask() à part.
[jimsmooth.sci_170517.pdf](https://github.com/jimlab/jimlab/files/1008452/jimsmooth.sci_170517.pdf)

SVP écrire des tests pour les différentes syntaxes de jimsmooth(). Cela facilitera vos développements.

**Comment by JimAmel** [comment]
le ralentissement de la fonction etait due à la fonction "cat" .elle fonctionne à présent

**Comment by sgougeon** [comment]
oui. Elle était inutile. Ceci dit, ce ralentissement paraissait anormal. Je verrai s'il y a lieu de poster un rapport de bug sur Bugzilla.
Une fois le code débogué, SVP supprimer les disp().

**Comment by JimAmel** [comment]
Image d'origine et image modifiée de même taille.ET les disp() supprimés.
